packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -211,10 +198,14 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
             // it will remember and not try to load anything from that path again (a `false` will be
             // put into the cache for that cache key). Instead, we force a new path, by adding precompiled
             // to the module path and use the absolute path to the module to load it.
+            logger.debug(`Successfully copied the precompiled build for ${opts.nativeModuleName} ${label}.`);
```

Comment:
S: I was wondering if its good to have a debug log for "copying from X to Y" as well in line 186?